src/tools/bin/tbg

```diff
@@ -423,8 +427,12 @@ fi
 start_dir=`dirname $0`
 
 if [ -d "$job_outDir" ] ; then
-    echo "job name already in use, can't create new folder"
-    exit 1
+    if [ "$force" = "true" ] ; then
```

Comment:
note: this is valid, but I usually prefer `==` to be more explicit

http://tldp.org/LDP/abs/html/comparison-ops.html

Reply:
Fixed. Amended.